docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Adds alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Adds alert suppression for EQL rules with non-sequence queries ({kibana-pull}176422[#176422]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now GA ({kibana-pull}180838[#180838]).
+* Allows you to add newly-created fields to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
```

Comment:
What do you think of this? I think this sounds good, but wonder if this gives a bit more context - 
```suggestion
* Allows you to add query defined fields to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
```